converters/converter.go

```diff
@@ -0,0 +1,22 @@
+package converters
+
+var converters = make(map[string]Converter)
+
+func init() {
+	register("postgres", &Pg{})
```

Comment:
Both PG and MySQL types are `struct{}` you should change the receivers to be values and those line would be just `register("asdf", T{})`